src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$main-color: #1d1d1d;
+$secondary-color: #fff;
```

Comment:
For landing you can create more variables

Reply:
I will add mixins and variables later.  Did not optimize it because I wanted to switch to landing 2 faster.